plugin.video.catchuptvandmore/resources/lib/simpleplugin.py

```diff
@@ -504,7 +504,7 @@ def initialize_gettext(self):
         :return: :meth:`Addon.gettext` method object
         :raises simpleplugin.SimplePluginError: if the addon's English :file:`strings.po` file is missing
         """
-        strings_po = os.path.join(self.path, 'resources', 'language', 'resource.language.en_gb', 'strings.po')
+        strings_po = os.path.join(self.path, 'resources', 'language', 'English', 'strings.po')
         if os.path.exists(strings_po):
```

Comment:
I doubt you're allowed to read the strings file directly / parse it / change it from your plugin. Why do you need this? The API has methods to retrieve the strings you need.